external/storm-solr/src/main/java/org/apache/storm/solr/config/SolrConfig.java

```diff
@@ -39,4 +40,13 @@ public SolrConfig(String zkHostString) {
     public String getZkHostString() {
         return zkHostString;
     }
+
+    public int getTickTupleInterval() {
+        return tickTupleInterval;
+    }
+
+    public void setTickTupleInterval(int tickTupleInterval) {
```

Comment:
@vesense Can you please delete this set method and pass the tickTupleInterval in the constructor, or use a builder pattern. This object should be immutable and fully initialized once passed to the Bolt.